docs/sources/clients/promtail/troubleshooting.md

```diff
@@ -19,6 +19,31 @@ To start Promtail in dry run mode use the flag `--dry-run` as shown in the examp
 cat my.log | promtail --stdin --dry-run --client.url http://127.0.0.1:3100/loki/api/v1/push
 ```
 
+## Inspecting
```

Comment:
```suggestion
## Inspecting log entry changes
```

Reply:
I think `Inspecting pipeline stages` will scan better here, any objections?

Reply:
You can change it.  When I read this header from the prospective of a not-advanced Loki user, I don't know or remember what pipeline stages are (in Loki).  I also think that we're not actually inspecting pipeline stages, but instead we're inspecting the results caused by invoked pipeline stages.  Does my explanation make sense?

Reply:
OK thanks; I think any Promtail user who's coming to this troubleshooting page will already know what pipeline stages, so I think that's acceptable.